### ISSUE TITLE
[COGNITION]: GIBCT Caution Flags - Non-native focus targets SHOULD NOT have the yellow focus halo

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9310]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The `<h3>` Alerts from VA and other federal agencies is a non-native focus target, so our best practice is not to include a yellow focus halo. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] When the H3 receives focus, it should not have a yellow focus halo
- [ ] No other functionality of the current implementation should be changed

## Environment

* https://staging.va.gov/gi-bill-comparison-tool/profile/14900416#viewWarnings

## WCAG or Vendor Guidance (optional)

* [CSS Tricks: Focus Styles on Non-Interactive Elements?](https://css-tricks.com/focus-styles-non-interactive-elements/)

## Screenshots or Trace Logs

![Screen Shot 2020-05-19 at 6 00 55 PM](https://user-images.githubusercontent.com/934879/82387116-c63dc780-99fb-11ea-8d13-1456f723f71d.png)

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12874

**Comment by sonntag-adam** [comment]
@1Copenut this is in staging ready for your review

**Comment by 1Copenut** [comment]
@sonntag-adam Verified in staging. Moving to close.
